src/components/Card/CardList/index.tsx

```diff
@@ -0,0 +1,10 @@
+import { ReactNode } from "react";
+import "./CardList.css";
+
+interface Props {
+  children: ReactNode;
+}
+
+export const CardList = ({ children }: Props) => {
```

Comment:
`PropsWithChildren`이라는 내장 타입이 있습니다. 따로 `children` 타입을 주지 않으셔도 됩니다. 
```suggestion
interface Props {
  children: ReactNode;
}

export const CardList = ({ children }: PropsWithChildren) => {
```